### DISCUSSION TITLE
v-touch-pan event with own params

### DISCUSSION THREAD

**Discussion by romkhram** [quasarframework/quasar#10803] [General - Components / Directives / etc] [answered]
v-touch-pan.prevent="handlePan( {what i need write this to use with own params?}, ownParams )"

to use in setup

    const handlePan = ({ evt, ...newInfo }, ownParams) => {
      console.log(ownParams);
    }

Thanks

**Comment by ontwikkelfabriek** [comment, marked as answer]
@romkhram 



```
 v-touch-pan.prevent.mouse="(details) => handlePan(details, {'foo': 'bar'})"
```
Here's a codepen.io:
https://codepen.io/ontwikkelfabriek/pen/jOweExP

**Reply by romkhram** [reply]
Thank you!

**Comment by RonchettiAssociati** [comment]
Thank you!
